src/period-calculation/monthly-periods/compute-monthly-index.ts

```diff
@@ -0,0 +1,98 @@
+import { FIXED_PERIOD_TYPES } from '../period-types'
+import { PeriodIdentifier } from '../types'
+import novemberQuarters from './quarters'
+
+type ComputeMonthlyIndex = (args: {
+    periodType: PeriodIdentifier
+    month: number
+}) => number
+const computeMonthlyIndex: ComputeMonthlyIndex = ({ periodType, month }) => {
```

Comment:
The reason for the existence of this function is that the index was computed with the loop in the `generateMonthlyPeriods` function, but now that the `buildMonthlyFixedPeriod` will have to work outside of the context of that loop (as it'll be used by other utility functions that won't use that loop), there needs to be a different way of computing the correct index

Reply:
Btw this is one of the causes for this breaking change:

> * With the current implementation, QuaterlyNovember doesn't work with the Ethiopian calendar